### ISSUE TITLE
I may have a way to move the BYOB functionality to Whiskers

### ISSUE THREAD

**Issue by Radical-from-Scratch** [NoMod-Programming/Whiskers-2.8#24]
Would ya like some help?

**Comment by NoMod-Programming** [comment]
Sure!

**Comment by Radical-from-Scratch** [comment]
Which version of the .image file should I work on?

**Comment by Jonathan50** [comment]
Lambda yay

**Comment by Radical-from-Scratch** [comment]
? What are you talking about

**Comment by Jonathan50** [comment]
Oh, you need to learn functional programming! It's so much fun and better than imperative programming!

**Comment by Jonathan50** [comment]
https://en.wikipedia.org/wiki/Functional_programming

**Comment by Radical-from-Scratch** [comment]
NOBODY LIKES A GRAMMAR NAZI!


But seriously, please have actual, meaningful discussion here instead of those posts.

**Comment by Jonathan50** [comment]
> NOBODY LIKES A GRAMMAR NAZI!

I wasn't talking about grammar, I was talking about terminology :stuck_out_tongue:
\(Wait.... I was just being a terminology nazi then. Oh well, I don't mind :package:)

> But seriously, please have actual, meaningful discussion here instead of those posts.

You asked me:
> ? What are you talking about

**Comment by Radical-from-Scratch** [comment]
Yeah, because I didn't think were talking about meaningful things, otherwise I would have understood you

**Comment by Firedrake969** [comment]
Lambdas are meaningful because BYOB has them.
Just because something's confusing doesn't mean it's irrelevant :P

**Comment by Radical-from-Scratch** [comment]
Eh, if you say so.
